modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,88 @@
+* PipelineResources can now be marked as optional.
+* Support for sharing data between Tasks using Task Results and Task Params.
+* Support to configure Pipelines to not overwrite the `HOME` environment variable and `workingDir` of Steps.
+* Similar to Task Steps, `sidecars` now support script mode.
+* You can now specify a different scheduler name in TaskRun `podTemplate`.
+* Support for Star Array Notation for variable substitution.
+* Tekton Controller can now be configured to monitor an individual namespace.
+* A new description field is now added to the specification of Pipeline, Task, ClusterTask, Resource and Condition.
+* Addition of  proxy parameters to Git PipelineResources.
+
+=== `tkn`CLI new features
+
+* The `describe` sub-command is now added for the following `tkn` resources -- `eventlistener`, `condition`, `triggertemplate`, `clustertask`, and `triggerbinding`.
+* Support added for `v1beta1` to the following commands along with backward comptibility for `v1alpha1` - `clustertask`, `task`, `pipeline`, `pipelinerun`,  and `taskrun`.
+* The following commands can now list from all namespaces using the `--all-namespaces` flag option:
+** `tkn task list`
+** `tkn pipeline list`
+** `tkn taskrun list`
+** `tkn pipelinerun list`
+The output of these commands is also enhanced to display information without headers using `--no-headers` flag option.
+* You can now start a Pipeline using default parameter values by specifying `--use-param-defaults` flag in the `tkn pipelines start` command.
+* Support for Workspace is now added to `tkn pipeline start` and `tkn task start` commands.
+* A new `clustertriggerbinding` command is now added with sub-commands - `describe`, `delete`, and `list`.
+* You can now directly start a pipeline run using a local or remote `yaml` file.
+* The `describe` sub-command now displays an enhanced and detailed output. With the addition of new fields,  such as `description`, `timeout`, `param description`, and `sidecar status`, the command output now provides more detailed information about a specific `tkn` resource.
+* The `tkn task log` command will now display logs directly if only one task is present in the namespace.
+
+=== Triggers new features
+* Triggers can now create both `v1alpha1` and `v1beta1` Pipeline resources.
+* Support for new Common Expression Language (CEL) interceptor function `compareSecrets`. This function securely comparing strings to secrets in CEL expressions.
+* Support for authentication and authorization at the EventListener Trigger level.
+
+== Deprecated features
+The following items are deprecated or not supported in release 1.0:
+
+* Environment variable `$HOME`, and variable `workingDir` of Steps specification will be changed in the next release. Currently in a Step container, `HOME` and  `workingDir` are overwritten to `/tekton/home` and `/workspace` respectively.
```

Comment:
The  Environment variable 
... in the specification for Steps will be changed...

Reply:
how about - in the Steps specification